lib/services/mihin_interface.dart

```diff
@@ -0,0 +1,110 @@
+import 'dart:io';
+
+import 'package:fhir/r4.dart';
+import 'package:fhir_at_rest/fhir_at_rest.dart' as rest;
+import 'package:fhir_auth/fhir_auth.dart';
+
+import 'credentials.dart';
+import 'db_interface.dart';
+
+class MihinInterface {
+  MihinInterface();
+
+  static Future uploadAllToMihin() async {
```

Comment:
Does this Future have/need a return type? Even `return true` to ensure that the method has completed.

Reply:
Same options as above. I print off the results to the command line, but I can return them instead.